build.sh

```diff
@@ -150,10 +150,10 @@ check_prereqs()
     hash cmake 2>/dev/null || { echo >&2 "Please install cmake before running this script"; exit 1; }
 
 
-    # Minimum required version of clang is version 3.9 for arm/armel cross build
+    # Minimum required version of clang is version 4.0 for arm/armel cross build
     if [[ $__CrossBuild == 1 && ("$__BuildArch" == "arm" || "$__BuildArch" == "armel") ]]; then
-        if ! [[ "$__ClangMajorVersion" -gt "3" || ( $__ClangMajorVersion == 3 && $__ClangMinorVersion == 9 ) ]]; then
-            echo "Please install clang3.9 or latest for arm/armel cross build"; exit 1;
+        if ! [[ "$__ClangMajorVersion" -ge "4"]]; then
```

Comment:
This will break the arm64 innerloop build because we are not using docker to build yet. I would propose keeping 3.9 and we can make this change later.

Reply:
Is `("$__BuildArch" == "arm" || "$__BuildArch" == "armel")` == true for arm64?

Reply:
Ah no that is correct; however, the armel docker image will need to be updated.